content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+After selecting the correct application, enter the Credential Name (Please see HPE GreenLake Cloud Console User Guide for supported characters).  Click the Create Credentials button to proceed with Client Credentials creation.
+
+![Create Credential no input yet](/img/create-credentials-menu.png "Generate Client Credentials 1st time")
+
+Once the Create Credential button is clicked, the following information about the OAuth (Open Authorization) will be shown. These are the Client ID and Client Secret information menu that is going to be shown only once. Please copy both Client ID and Client Secret to a safely recorded location. In the case that the user missed to copy these information, then the user can only regenerate the Client Secret at later time using the Client-Secret Reset menu.
+
+![](/img/api-client-credential-created.png "Credentials Created Close")
+
+After closing the credential creation menu, the user can observe the prior created API Client Credentials by identifying the Credential Name.
+
+![](/img/application-credential-created-prior-shown.png "API Client Credentials are created")
+
+Clicking on the down arrow, user can generate the access token required to use the API.
+
+![](/img/api-client-credential-get-access-token.png "Time to obtain the Access Token")
+
+After clicking on the Generate Access Tokens button, the menu requires user to enter the Client Secret obtained from the API Client Credentials. This is the time to copy and paste the client secret from Credential Created menu so that the user can obtain the access token.
+
+![](/img/generate-access-token-with-secret.png "Use the client secret to generate Access Token")
+
+Copy the access token using using the "sheets" icon and store this access token to a safely recorded location.
+
+![](/img/access-token-created-and-close.png "Access Token Generated and Consumed")
+
+Afterward, the user can embed the access token to the REST API request in order to perform the HTTP method against the desired resource in order to obtain the response.  Note that the user must use the correct base-URL according to the region where the DSCC is deployed. Currently these are the base-URL and the corresponding region where the DSCC is deployed (November 2021)
+
+| DSCC Region  | base-URL                       |
+| ------------ | ------------------------------ |
+| EU Central   | https://eu1.data.cloud.hpe.com |
+| AP Northeast | https://jp1.data.cloud.hpe.com |
+| US West      | https://us1.data.cloud.hpe.com |
+
+#### Oops! What if I missed to copy the client secret for this instance of client credential?
+
+The client secret can be recreated inside the create credentials menu by clicking on the three dots at the bottom menu. Once the three dot menu opened, the user must click on the Reset Client Secret menu. This prior action will open the Credentials Created menu to show the new client secret for one time only. 
```

Comment:
Change "client secret" to "Client Secret" for consistency at the beginning of the paragraph